Assets/Scripts/Throw/NextColor.cs

```diff
@@ -63,4 +72,20 @@ private ColorSO GetRandomColorSO()
     {
         return _gameManager.GetRandomColorSO();
     }
+
+    private ColorSO ThreeColors()
+    {
+   
+        if( _gameManager._ColorChangeRand==true)
+              return _gameManager.GetRandomColorSO();
+        else
+        {
+            if (colorindex == 3)
```

Comment:
Don't hard code the number `3` here, since we might have more colors later on and the tutorial only has one color. This could lead to bugs later on. You can use `_gameManager.NumberOfColors` to get the number of color indices.